src/sys/node/public.ts

```diff
@@ -3,19 +3,25 @@ import type { CompilerSystem, Logger } from '@stencil/core/internal';
 /**
  * Creates a "logger", based off of NodeJS APIs, that will be used by the compiler and dev-server.
  * The NodeJS "process" object must be provided as a property in the first argument's object.
- * @param c an object containing a `Process` for Stencil to use
  * @returns a {@link Logger} object
  */
-export declare function createNodeLogger(c: { process: any }): Logger;
+export declare function createNodeLogger(): Logger;
```

Comment:
If we do move forward with this change, we should update https://github.com/ionic-team/stencil/blob/b91f94a122dc9add876f0277f99afc92c96e66dd/scripts/test/validate-build.ts#L267 as well